src/main/java/com/sonyericsson/jenkins/plugins/bfa/tokens/Renderer.java

```diff
@@ -133,7 +146,11 @@ public String render(FailureCauseBuildAction action) {
         final FailureCauseDisplayData data = action.getFailureCauseDisplayData();
         if (data.getFoundFailureCauses().isEmpty() && data.getDownstreamFailureCauses().isEmpty()) {
             logger.info("there were no causes");
-            return noFailureText;
+            if (!useHtmlFormat && escapeHtml) {

```

Comment:
Would it make sense to use `!useHtmlFormat || escapeHtml` instead? Or maybe get rid of `escapeHtml` completely and just use `useHtmlFormat` to control escaping? Or in other words, if I set `escapeHtml` to `true`, would it make sense to set  `useHtmlFormat` to `true`? Or would that be a pointless configuration? Same for setting `escapeHtml` to `false` and `useHtmlFormat` to `false`.

(Maybe I'm just confused about the name `useHtmlFormat`.)

Reply:
`escapeHtml` is the parameter that I can't get rid of, it's now inherited from the parent class. `useHtmlFormat` is the one that has always been there and if it is true it will escape all relevant parts and always has. So the "new" parameter is then only relevant if the user wants a non html formatted list.